drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -849,7 +849,11 @@ void RasterizerCanvasGLES3::_record_item_commands(const Item *p_item, RID p_rend
 
 		if (c->type != Item::Command::TYPE_MESH) {
 			// For Meshes, this gets updated below.
-			_update_transform_2d_to_mat2x3(base_transform * draw_transform, state.instance_data_array[r_index].world);
+			Transform2D final_transform = base_transform * draw_transform;
+			final_transform.set_origin(
+					Vector2(
+							Math::round(final_transform.get_origin().x), Math::round(final_transform.get_origin().y)));
```

Comment:
```suggestion
			final_transform.set_origin(final_transform.get_origin().round());
```
Dedicated method for this 🙂